src/main/java/org/isf/accounting/rest/BillController.java

```diff
@@ -123,13 +123,13 @@ public ResponseEntity<FullBillDTO> newBill(@RequestBody FullBillDTO newBillDto)
 		if (pat != null) {
 			bill.setBillPatient(pat);
 		} else {
-			throw new OHAPIException(new OHExceptionMessage("Patient not found."));
+			return ResponseEntity.badRequest().body(new OHExceptionMessage("Patient not found."));
```

Comment:
I find three different return values for something not found:

<pre>
return ResponseEntity.badRequest().body("XXXX not found.");
return ResponseEntity.badRequest().body(new OHExceptionMessage("XXXX not found."));
return ResponseEntity.notFound().build();
</pre>

Shouldn't there only be one form?